app/assets/stylesheets/pages/_home.scss

```diff
@@ -15,3 +20,11 @@
     border-radius: 0;
   }
 }
+
+h4 {
```

Comment:
don't use plain h4 as selector, give the tagline an id="tagline" or subtitle, and then select that. Don't want to affect the other h4s in case, and id has higher priority so it does not get overwritten by other h4 selectors in our scss